info.yml

```diff
@@ -16,4 +16,4 @@ description: "A config for Unity-based pipelines. Based on tk-config-default2."
 
 # Required minimum versions for this item to run
 requires_shotgun_version: "v7.2.0"
-requires_core_version: "v0.19.6"
+requires_core_version: "v0.19.5"
```

Comment:
Is this right, we're downgrading the required core version?

Reply:
0.19.5 is what is used in tk-config-default2. Also this is the minimum required version. Actual version used is v0.19.17 (in core/core_api.yml) if I understood correctly.

Reply:
Leave it like that (`v0.19.5`)

As you say the actual tk-core version is `v0.19.7`, and studios can decide to change that. But if they want to use `tk-config-unity` as their pipeline config they will probably fail if they use something lower than `v0.19.5`. So they will edit `info.yml` to reduce the number and I am pretty sure everything will be fine :-)